deploy.sh

```diff
@@ -0,0 +1,36 @@
+if ([ $TRAVIS_PULL_REQUEST == "false" ])
+then
+    #Clone the latest gh-pages
+    git clone https://github.com/ScottLogic/StockFlux.git --branch gh-pages gh-pages
+
+    #Get line with version from the file -> get the second word -> remove brackets
+    VERSION=$(grep "version" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    #Get line with the release type (develop/master) from the file -> get the second word -> remove brackets
+    TYPE=$(grep "type" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    rm -rf ./gh-pages/$TYPE
+    cp -r ./public ./gh-pages/$TYPE
+    rm -rf ./gh-pages/$VERSION
```

Comment:
@WPFerg @akhmorgan @mbssantos @mic-smith third opinion needed.

I will just remind one more thing- this will only ever happen after the PR is merged, never before- it needs more than one person to make a mistake.